Assets/Scripts/Combat/Combantants/EnemyCombatant.cs

```diff
@@ -67,11 +70,13 @@ protected override void TakeTurnWhileDead()
     protected override void TakeTurnWhileAlive()
     {
         DisplayUI();
-        enemyUI.GetComponent<Text>().text = "I Attak! But actually i slep";
+        EnemyUI.GetComponent<Text>().text = "I Attak!";
 
-        StartCoroutine(SimpleEnemyAttack());
+        Debug.Log(Abilities.Length);
+        Abilities[Random.Range(0, Abilities.Length - 1)].StartAbility(false);
     }
 
+    // Deprecated
     IEnumerator SimpleEnemyAttack()
     {
         yield return new WaitForSeconds(2);
```

Comment:
I'm guessing if you haven't removed it yet it's cause it's still used for testing other stuff?

Reply:
I figured it might be still used for the occasional test, at least until we work most of the inevitable combat bugs. Currently, enemies only have access to 1 other ability: Bonk. Having a quicker, more reliable option for debugging could have its uses for now